opensearch/services/datasets/datasets.php

```diff
@@ -72,6 +72,22 @@ private function getDCOAuthorsByDataset($dataset) {
 		return $this->sparqlSelect($query);
 	}
 
+	/*
```

Comment:
Does this mean we have yet another result set where we're doing a query per result to get the data types? What is the performance of the result set now?